src/hooks/useAuth.jsx

```diff
@@ -0,0 +1,9 @@
+import { useAuthContext } from "../contexts/userAuthContext";
```

Comment:
It's so that I can switch between the login and signup components. The button used to switch is present inside the component itself and I need the value outside, which is why I used context here.